.github/workflows/validation-jobs.yml

```diff
@@ -24,14 +24,12 @@ jobs:
     timeout-minutes: 30
     steps:
       - uses: actions/checkout@v4
-
       - uses: dtolnay/rust-toolchain@stable
-
```

Comment:
in this part of the file every steps of jobs are separated by empty new line, could you keep them? or remove them all from the iOS and android jobs